frontend/src/scenes/max/components/HedgehogAvatar.tsx

```diff
@@ -0,0 +1,75 @@
+import { IconSparkles } from '@posthog/icons'
+import { Tooltip } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { HedgehogActor, HedgehogBuddy } from 'lib/components/HedgehogBuddy/HedgehogBuddy'
+import { useEffect, useRef } from 'react'
+import { userLogic } from 'scenes/userLogic'
+
+interface HedgehogAvatarProps {
+    onExpand: () => void
+    waveInterval?: number
+    isExpanded: boolean
+}
+
+export function HedgehogAvatar({ onExpand, waveInterval = 5000, isExpanded }: HedgehogAvatarProps): JSX.Element {
+    const { user } = useValues(userLogic)
+    const hedgehogActorRef = useRef<HedgehogActor | null>(null)
+
+    // Trigger wave animation periodically when collapsed
+    useEffect(() => {
+        let interval: ReturnType<typeof setInterval> | null = null
+
+        if (!isExpanded && hedgehogActorRef.current) {
+            interval = setInterval(() => {
+                hedgehogActorRef.current?.setAnimation('wave')
+            }, waveInterval)
+        }
+
+        return () => {
+            if (interval) {
+                clearInterval(interval)
+            }
+        }
+    }, [isExpanded, waveInterval])
+
+    return (
+        <div className="relative flex items-center justify-end mb-2 mr-4">
+            <Tooltip
+                title={
+                    <>
+                        <IconSparkles className="mr-1.5" />
+                        Max AI - Create insights, talk to your data, and more
+                    </>
+                }
+                placement="top-start"
+                delayMs={0}
+            >
+                <div
+                    className="size-10 rounded-full overflow-hidden border border-border-primary shadow-lg hover:shadow-xl transition-all duration-200 cursor-pointer -scale-x-100 hover:scale-y-110 hover:-scale-x-110 flex items-center justify-center bg-bg-light"
```

Comment:
These shadows are pretty un-PostHoggy, I propose we give him the same padding with translucent background as the actual input overlay (and no shadow)
![A - over](https://github.com/user-attachments/assets/ba008bda-39dc-45f6-8f8c-509245c04792)